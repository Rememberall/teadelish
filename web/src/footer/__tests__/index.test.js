import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';

import Footer from '../index';

it('should not change', () => {
  const component = (
    <MemoryRouter>
      <Footer />
    </MemoryRouter>
  );
  const tree = renderer.create(component).toJSON();
  expect(tree).toMatchSnapshot();
});
