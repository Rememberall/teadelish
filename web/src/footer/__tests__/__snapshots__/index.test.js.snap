// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should not change 1`] = `
<footer>
  <input
    className="search"
    placeholder="Search for teas, venues"
    type="text"
  />
  <nav>
    <ul>
      <li
        className="selected"
      >
        <a
          href="/"
          onClick={[Function]}
        >
          Home
        </a>
      </li>
      <li>
        <a
          href="/checkin"
          onClick={[Function]}
        >
          Checkin
        </a>
      </li>
    </ul>
  </nav>
</footer>
`;
